develop/parachains/testing/benchmarking.md

```diff
@@ -1,9 +1,9 @@
 ---
```

Comment:
Yes, good correcting. Benchmarking is not really "_testing_".

Good resource to add to the page: https://paritytech.github.io/polkadot-sdk/master/polkadot_sdk_docs/reference_docs/frame_benchmarking_weight/index.html